pkg/client/monitoring/v1alpha1/types.go

```diff
@@ -125,6 +127,16 @@ type PrometheusStatus struct {
 	UnavailableReplicas int32 `json:"unavailableReplicas"`
 }
 
+// LogSpec defines the configured log messages from the Prometheus logger.
+type LogSpec struct {
```

Comment:
I'd like to wait for @fabxc to return from vacation, as he has a much better idea on how the logging story is going to continue with Prometheus 2.0.

Reply:
Hi, sorry to post here but just one question: it seems there is no way to separate TRACE/DEBUG/INFO/WARN on the stdout and ERROR on the stderr, am I right?

And if yes, could you explain why please :) ? Thanks

Reply:
It’s not possible as it is currently written. Typically, it’s good practice to have all logs for a service be ingested by your logging solution (journald, or some custom logging and processing stack) and then filter/process those as you wish. It’s not really feasible for the operator to encode all possible fancy processing capabilities and preferable for the user to create a custom pipeline.

Reply:
@sneko I believe what you are talking about is the logs from the Prometheus Operator, unfortunately those come from kubernetes/client-go and they use a global logger, that we have little control over. You can however still control the log level of that one (see the [glog documentation](https://github.com/golang/glog)).